tests/attention/test_cudnn_prefill_paged_cu_seqlens.py

```
@@ -0,0 +1,149 @@
+    The shipping gate is a version compare, but the FE package version is an
+    unreliable proxy on dev builds, so tests probe the real capability instead.
+    """
+    if not cudnn_prefill.CUDNN_AVAILABLE or not torch.cuda.is_available():
+        return False
+    try:
+        import cudnn
+
+        return cudnn.backend_version() >= 92500 and hasattr(
+            cudnn, "attention_implementation"
+        )
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)